trivy.yaml

```diff
@@ -0,0 +1,4 @@
+exit-code: 0
```

Comment:
Same config as the one in Indico charm repo. We need to discuss in the future whether we drive trivy configuration here or in CI/CD, as both don't work well. For instance in CI/CD trivy would filter out anything not WARNING or CRITICAL, but that behavior is overridden here and trivy would also report HIGH, MINOR etc.